pkg/perf/iperf_types.go

```diff
@@ -0,0 +1,134 @@
+package perf
```

Comment:
What is the point of having these here while they exist in go-iperf code?

Reply:
because go-iperf creates a temp directory in `init` so by importing go-iperf it would cause the same problem we are avoiding each time the task runs